packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -103,6 +103,7 @@ function FlowsheetDataGrid(props) {
         columnHeaderHeight={columnHeaderHeight}
         onCellSelect={onCellSelect}
         onClearSelectedCells={onClearSelectedCells}
+        hasNoResultCells
```

Comment:
The Flowsheet should be creating the no result cells.  There should not be any changes required at another level.

Reply:
Done